--- a/components/signin/public/base/signin_buildflags.gni
+++ b/components/signin/public/base/signin_buildflags.gni
@@ -1,5 +1,5 @@
-declare_args() {
-  enable_signin = false
-  enable_dice_support = false
-
-  google_build = false
-}
+declare_args() {
+  enable_signin = true
+  enable_dice_support = true
+
+  google_build = true
+}
--- a/components/signin/public/base/signin_pref_names.h
+++ b/components/signin/public/base/signin_pref_names.h
@@ -5,6 +5,8 @@
 #ifndef COMPONENTS_SIGNIN_PUBLIC_BASE_SIGNIN_PREF_NAMES_H_
 #define COMPONENTS_SIGNIN_PUBLIC_BASE_SIGNIN_PREF_NAMES_H_
 
+// Google Sign-In preferences for password and data sync
+
 namespace prefs {
 
 // Boolean that indicates whether the user is allowed to sign in to Helium.