--- a/chrome/app/chrome_version.rc.version
+++ b/chrome/app/chrome_version.rc.version
@@ -10,7 +10,7 @@
 //
 
 VS_VERSION_INFO VERSIONINFO
- FILEVERSION @MAJOR@,@MINOR@,@BUILD@,@PATCH@
+ FILEVERSION @HELIUM_MAJOR@,@HELIUM_MINOR@,@HELIUM_PATCH@,@HELIUM_PLATFORM@
  PRODUCTVERSION @MAJOR@,@MINOR@,@BUILD@,@PATCH@
  FILEFLAGSMASK 0x17L
 #ifdef _DEBUG
@@ -28,7 +28,7 @@ BEGIN
         BEGIN
             VALUE "CompanyName", "@COMPANY_FULLNAME@"
             VALUE "FileDescription", "@PRODUCT_FULLNAME@"
-            VALUE "FileVersion", "@MAJOR@.@MINOR@.@BUILD@.@PATCH@"
+            VALUE "FileVersion", "@HELIUM_MAJOR@.@HELIUM_MINOR@.@HELIUM_PATCH@.@HELIUM_PLATFORM@"
             VALUE "InternalName", "@INTERNAL_NAME@"
             VALUE "LegalCopyright", "@COPYRIGHT@"
             VALUE "OriginalFilename", "@ORIGINAL_FILENAME@"
--- a/chrome/app/main_dll_loader_win.cc
+++ b/chrome/app/main_dll_loader_win.cc
@@ -84,7 +84,7 @@ base::FilePath GetModulePath(std::wstrin
   // executable's directory and return the path if it can be read. This is the
   // expected location of modules for proper installs.
   const base::FilePath module_path =
-      exe_dir.AppendASCII(chrome::kChromeVersion).Append(module_name);
+      exe_dir.AppendASCII(chrome::kHeliumVersion).Append(module_name);
   if (ModuleCanBeRead(module_path))
     return module_path;
 
--- a/chrome/app/version_assembly/BUILD.gn
+++ b/chrome/app/version_assembly/BUILD.gn
@@ -40,6 +40,6 @@ windows_manifest("chrome_exe_manifest")
 process_version("version_assembly_manifest") {
   template_file = "version_assembly_manifest.template"
   sources = [ "//chrome/VERSION" ]
-  output = "$root_build_dir/$chrome_version_full.manifest"
+  output = "$root_build_dir/$helium_version_full.manifest"
   process_only = true
 }
--- a/chrome/app/version_assembly/chrome_exe_manifest.template
+++ b/chrome/app/version_assembly/chrome_exe_manifest.template
@@ -3,8 +3,8 @@
   <dependency>
   <dependentAssembly>
     <assemblyIdentity type='win32'
-      name='@MAJOR@.@MINOR@.@BUILD@.@PATCH@'
-      version='@MAJOR@.@MINOR@.@BUILD@.@PATCH@' language='*'/>
+      name='@HELIUM_MAJOR@.@HELIUM_MINOR@.@HELIUM_PATCH@.@HELIUM_PLATFORM@'
+      version='@HELIUM_MAJOR@.@HELIUM_MINOR@.@HELIUM_PATCH@.@HELIUM_PLATFORM@' language='*'/>
   </dependentAssembly>
   </dependency>
 </assembly>
\ No newline at end of file
--- a/chrome/app/version_assembly/version_assembly_manifest.template
+++ b/chrome/app/version_assembly/version_assembly_manifest.template
@@ -1,8 +1,8 @@
 <assembly
   xmlns='urn:schemas-microsoft-com:asm.v1' manifestVersion='1.0'>
   <assemblyIdentity
-      name='@MAJOR@.@MINOR@.@BUILD@.@PATCH@'
-      version='@MAJOR@.@MINOR@.@BUILD@.@PATCH@'
+      name='@HELIUM_MAJOR@.@HELIUM_MINOR@.@HELIUM_PATCH@.@HELIUM_PLATFORM@'
+      version='@HELIUM_MAJOR@.@HELIUM_MINOR@.@HELIUM_PATCH@.@HELIUM_PLATFORM@'
       type='win32'/>
   <file name='chrome_elf.dll'/>
 </assembly>
--- a/chrome/browser/chrome_content_browser_client.cc
+++ b/chrome/browser/chrome_content_browser_client.cc
@@ -1301,7 +1301,7 @@ base::FilePath GetModulePath(std::wstrin
   // executable's directory and return the path if it can be read. This is the
   // expected location of modules for proper installs.
   const base::FilePath module_path =
-      exe_dir.AppendASCII(chrome::kChromeVersion).Append(module_name);
+      exe_dir.AppendASCII(chrome::kHeliumVersion).Append(module_name);
   if (base::PathExists(module_path)) {
     return module_path;
   }
--- a/chrome/browser/web_applications/chrome_pwa_launcher/chrome_pwa_launcher_util.cc
+++ b/chrome/browser/web_applications/chrome_pwa_launcher/chrome_pwa_launcher_util.cc
@@ -21,7 +21,7 @@ base::FilePath GetChromePwaLauncherPath(
   base::FilePath chrome_dir;
   if (!base::PathService::Get(base::DIR_EXE, &chrome_dir))
     return base::FilePath();
-  base::FilePath launcher_path = chrome_dir.AppendASCII(chrome::kChromeVersion)
+  base::FilePath launcher_path = chrome_dir.AppendASCII(chrome::kHeliumVersion)
                                      .Append(kChromePwaLauncherExecutable);
   if (base::PathExists(launcher_path))
     return launcher_path;
--- a/chrome/common/chrome_constants.cc
+++ b/chrome/common/chrome_constants.cc
@@ -12,6 +12,7 @@
 namespace chrome {
 
 const char kChromeVersion[] = CHROME_VERSION_STRING;
+const char kHeliumVersion[] = HELIUM_VERSION_STRING;
 
 // The following should not be used for UI strings; they are meant
 // for system strings only. UI changes should be made in the GRD.
--- a/chrome/common/chrome_constants.h
+++ b/chrome/common/chrome_constants.h
@@ -15,6 +15,7 @@
 namespace chrome {
 
 extern const char kChromeVersion[];
+extern const char kHeliumVersion[];
 extern const base::FilePath::CharType kBrowserProcessExecutableName[];
 extern const base::FilePath::CharType kHelperProcessExecutableName[];
 extern const base::FilePath::CharType kBrowserProcessExecutablePath[];
--- a/chrome/common/chrome_version.h.in
+++ b/chrome/common/chrome_version.h.in
@@ -14,6 +14,9 @@
 #define CHROME_VERSION_BUILD @BUILD@
 #define CHROME_VERSION_PATCH @PATCH@
 
+#define HELIUM_VERSION_STRING \
+  "@HELIUM_MAJOR@.@HELIUM_MINOR@.@HELIUM_PATCH@.@HELIUM_PLATFORM@"
+
 // Branding Information
 
 #define COMPANY_FULLNAME_STRING "@COMPANY_FULLNAME@"
--- a/chrome/install_static/chromium_install_modes.h
+++ b/chrome/install_static/chromium_install_modes.h
@@ -17,14 +17,14 @@ namespace install_static {
 
 // The brand-specific company name to be included as a component of the install
 // and user data directory paths. May be empty if no such dir is to be used.
-inline constexpr wchar_t kCompanyPathName[] = L"";
+inline constexpr wchar_t kCompanyPathName[] = L"imput";
 
 // The brand-specific product name to be included as a component of the install
 // and user data directory paths.
-inline constexpr wchar_t kProductPathName[] = L"Chromium";
+inline constexpr wchar_t kProductPathName[] = L"Helium";
 
 // The brand-specific safe browsing client name.
-inline constexpr char kSafeBrowsingName[] = "chromium";
+inline constexpr char kSafeBrowsingName[] = "helium";
 
 // Note: This list of indices must be kept in sync with the brand-specific
 // resource strings in chrome/installer/util/prebuild/create_string_rc.
@@ -44,38 +44,38 @@ inline constexpr auto kInstallModes = st
             L"",  // Empty install_suffix for the primary install mode.
         .logo_suffix = L"",  // No logo suffix for the primary install mode.
         .app_guid =
-            L"",  // Empty app_guid since no integration with Google Update.
-        .base_app_name = L"Chromium",              // A distinct base_app_name.
-        .base_app_id = L"Chromium",                // A distinct base_app_id.
-        .browser_prog_id_prefix = L"ChromiumHTM",  // Browser ProgID prefix.
+            L"{FB68A146-637A-48C2-A0C4-1565DE45FEBD}",
+        .base_app_name = L"Helium",              // A distinct base_app_name.
+        .base_app_id = L"Helium",                // A distinct base_app_id.
+        .browser_prog_id_prefix = L"HeliumHTM",  // Browser ProgID prefix.
         .browser_prog_id_description =
-            L"Chromium HTML Document",  // Browser ProgID description.
-        .direct_launch_url_scheme = "chromium",
-        .pdf_prog_id_prefix = L"ChromiumPDF",  // PDF ProgID prefix.
+            L"Helium HTML Document",  // Browser ProgID description.
+        .direct_launch_url_scheme = "helium",
+        .pdf_prog_id_prefix = L"HeliumPDF",  // PDF ProgID prefix.
         .pdf_prog_id_description =
-            L"Chromium PDF Document",  // PDF ProgID description.
+            L"Helium PDF Document",  // PDF ProgID description.
         .active_setup_guid =
-            L"{7D2B3E1D-D096-4594-9D8F-A6667F12E0AC}",  // Active Setup
+            L"{FB68A146-637A-48C2-A0C4-1565DE45FEBD}",  // Active Setup
                                                         // GUID.
         .legacy_command_execute_clsid =
-            L"{A2DF06F9-A21A-44A8-8A99-8B9C84F29160}",  // CommandExecuteImpl
+            L"{729D2994-A2D0-4A62-84F2-28105C0A3B83}",  // CommandExecuteImpl
                                                         // CLSID.
-        .toast_activator_clsid = {0x635EFA6F,
+        .toast_activator_clsid = {0xE9852F6C,
                                   0x08D6,
                                   0x4EC9,
                                   {0xBD, 0x14, 0x8A, 0x0F, 0xDE, 0x97, 0x51,
                                    0x59}},  // Toast Activator CLSID.
-        .elevator_clsid = {0xD133B120,
+        .elevator_clsid = {0x140DE20F,
                            0x6DB4,
                            0x4D6B,
                            {0x8B, 0xFE, 0x83, 0xBF, 0x8C, 0xA1, 0xB1,
                             0xB0}},  // Elevator CLSID.
-        .elevator_iid = {0xbb19a0e5,
+        .elevator_iid = {0xC89A04FF,
                          0xc6,
                          0x4966,
                          {0x94, 0xb2, 0x5a, 0xfe, 0xc6, 0xfe, 0xd9,
                           0x3a}},  // IElevator IID and TypeLib
-        // {BB19A0E5-00C6-4966-94B2-5AFEC6FED93A}.
+        // {C89A04FF-8825-4629-B83E-77CC67D9CEED}.
         .tracing_service_clsid = {0x83f69367,
                                   0x442d,
                                   0x447f,
--- a/chrome/installer/mini_installer/mini_installer_constants.cc
+++ b/chrome/installer/mini_installer/mini_installer_constants.cc
@@ -12,7 +12,7 @@ namespace mini_installer {
 // The target name of the installer extracted from resources.
 const wchar_t kSetupExe[] = L"setup.exe";
 // The prefix of the chrome archive resource.
-const wchar_t kChromeArchivePrefix[] = L"chrome";
+const wchar_t kChromeArchivePrefix[] = L"helium";
 // The prefix of the installer resource.
 const wchar_t kSetupPrefix[] = L"setup";
 
--- a/chrome/installer/mini_installer/mini_installer_exe_version.rc.version
+++ b/chrome/installer/mini_installer/mini_installer_exe_version.rc.version
@@ -8,7 +8,7 @@
 // SDK versions which causes headaches building in some environments. The
 // VERSIONINFO resource will always be at index 1.
 1 VERSIONINFO
- FILEVERSION @MAJOR@,@MINOR@,@BUILD@,@PATCH@
+ FILEVERSION @HELIUM_MAJOR@,@HELIUM_MINOR@,@HELIUM_PATCH@,@HELIUM_PLATFORM@
  PRODUCTVERSION @MAJOR@,@MINOR@,@BUILD@,@PATCH@
  FILEFLAGSMASK 0x17L
 #ifdef _DEBUG
@@ -26,7 +26,7 @@ BEGIN
         BEGIN
             VALUE "CompanyName", "@COMPANY_FULLNAME@"
             VALUE "FileDescription", "@PRODUCT_INSTALLER_FULLNAME@"
-            VALUE "FileVersion", "@MAJOR@.@MINOR@.@BUILD@.@PATCH@"
+            VALUE "FileVersion", "@HELIUM_MAJOR@.@HELIUM_MINOR@.@HELIUM_PATCH@.@HELIUM_PLATFORM@"
             VALUE "InternalName", "mini_installer"
             VALUE "LegalCopyright", "@COPYRIGHT@"
             VALUE "ProductName", "@PRODUCT_INSTALLER_FULLNAME@"
--- a/chrome/installer/setup/BUILD.gn
+++ b/chrome/installer/setup/BUILD.gn
@@ -114,6 +114,7 @@ if (is_win) {
       "//components/crash/core/common",
       "//components/metrics:client_info",
       "//components/metrics:metrics_pref_names",
+      "//base/version_info",
       "//components/version_info:channel",
       "//third_party/crashpad/crashpad/client",
       "//third_party/crashpad/crashpad/util",
--- a/chrome/installer/setup/install_worker.cc
+++ b/chrome/installer/setup/install_worker.cc
@@ -34,6 +34,7 @@
 #include "base/strings/utf_string_conversions.h"
 #include "base/version.h"
 #include "base/version_info/channel.h"
+#include "base/version_info/version_info.h"
 #include "base/win/registry.h"
 #include "base/win/security_util.h"
 #include "base/win/sid.h"
@@ -583,10 +584,11 @@ void AddUninstallShortcutWorkItems(const
                                          InstallUtil::GetPublisherName(), true);
     install_list->AddSetRegValueWorkItem(
         reg_root, uninstall_reg, KEY_WOW64_32KEY, L"Version",
-        ASCIIToWide(new_version.GetString()), true);
+        ASCIIToWide(version_info::GetHeliumVersionNumber()), true);
     install_list->AddSetRegValueWorkItem(
         reg_root, uninstall_reg, KEY_WOW64_32KEY, L"DisplayVersion",
-        ASCIIToWide(new_version.GetString()), true);
+        ASCIIToWide(version_info::GetHeliumVersionNumber()),
+        true);
     // TODO(wfh): Ensure that this value is preserved in the 64-bit hive when
     // 64-bit installs place the uninstall information into the 64-bit registry.
     install_list->AddSetRegValueWorkItem(reg_root, uninstall_reg,
--- a/chrome/installer/setup/installer_crash_reporter_client.cc
+++ b/chrome/installer/setup/installer_crash_reporter_client.cc
@@ -103,7 +103,7 @@ bool InstallerCrashReporterClient::Repor
       L"SOFTWARE\\Policies\\Google\\Chrome";
 #else
   static const wchar_t kRegistryChromePolicyKey[] =
-      L"SOFTWARE\\Policies\\Chromium";
+      L"SOFTWARE\\Policies\\Helium";
 #endif
   static const wchar_t kMetricsReportingEnabled[] = L"MetricsReportingEnabled";
 
--- a/chrome/installer/setup/setup_constants.cc
+++ b/chrome/installer/setup/setup_constants.cc
@@ -7,14 +7,14 @@
 namespace installer {
 
 // Elements that make up install paths.
-const wchar_t kChromeArchive[] = L"chrome.7z";
-const wchar_t kChromeCompressedArchive[] = L"chrome.packed.7z";
+const wchar_t kChromeArchive[] = L"helium.7z";
+const wchar_t kChromeCompressedArchive[] = L"helium.packed.7z";
 const char kVisualElements[] = "VisualElements";
 const wchar_t kVisualElementsManifest[] = L"chrome.VisualElementsManifest.xml";
 
 // Sub directory of install source package under install temporary directory.
 const wchar_t kInstallSourceDir[] = L"source";
-const wchar_t kInstallSourceChromeDir[] = L"Chrome-bin";
+const wchar_t kInstallSourceChromeDir[] = L"Helium-bin";
 
 const wchar_t kMediaPlayerRegPath[] =
     L"Software\\Microsoft\\MediaPlayer\\ShimInclusionList";
--- a/chrome/installer/setup/setup_main.cc
+++ b/chrome/installer/setup/setup_main.cc
@@ -478,7 +478,7 @@ installer::InstallStatus RenameChromeExe
                                     WorkItem::MoveTreeOptions{});
 
   AddFinalizeUpdateWorkItems(original_state,
-                             base::Version(chrome::kChromeVersion),
+                             base::Version(chrome::kHeliumVersion),
                              *installer_state, setup_exe, install_list.get());
 
   // Add work items to delete Chrome's "opv", "cpv", and "cmd" values.
--- a/chrome/installer/util/logging_installer.cc
+++ b/chrome/installer/util/logging_installer.cc
@@ -126,7 +126,7 @@ base::FilePath GetLogFilePath(const inst
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
       FILE_PATH_LITERAL("chrome_installer.log");
 #else  // BUILDFLAG(CHROMIUM_BRANDING)
-      FILE_PATH_LITERAL("chromium_installer.log");
+      FILE_PATH_LITERAL("helium_installer.log");
 #endif
 
   // Fallback to current directory if getting the secure or temp directory
--- a/chrome/tools/build/win/create_installer_archive.py
+++ b/chrome/tools/build/win/create_installer_archive.py
@@ -25,7 +25,7 @@ ARCHIVE_DIR = "installer_archive"
 # suffix to uncompressed full archive file, appended to options.output_name
 ARCHIVE_SUFFIX = ".7z"
 BSDIFF_EXEC = "bsdiff.exe"
-CHROME_DIR = "Chrome-bin"
+CHROME_DIR = "Helium-bin"
 CHROME_PATCH_FILE_SUFFIX = "_patch"  # prefixed by options.output_name
 
 # compressed full archive suffix, will be prefixed by options.output_name
@@ -47,24 +47,18 @@ def BuildVersion():
     """Returns the full build version string constructed from information in
     VERSION_FILE.  Any segment not found in that file will default to '0'.
     """
-    major = 0
-    minor = 0
-    build = 0
-    patch = 0
+    values = {}
     # The version file is located at the directory ${CHROMIUM_SRC}/chrome.
     version_file_path = os.path.join(os.path.dirname(__file__), '../../..',
                                      VERSION_FILE)
     for line in open(version_file_path, 'r'):
-        line = line.rstrip()
-        if line.startswith('MAJOR='):
-            major = line[6:]
-        elif line.startswith('MINOR='):
-            minor = line[6:]
-        elif line.startswith('BUILD='):
-            build = line[6:]
-        elif line.startswith('PATCH='):
-            patch = line[6:]
-    return '%s.%s.%s.%s' % (major, minor, build, patch)
+        key, sep, value = line.rstrip().partition('=')
+        if sep:
+            values[key] = value
+    return '%s.%s.%s.%s' % (values.get('HELIUM_MAJOR', '0'),
+                            values.get('HELIUM_MINOR', '0'),
+                            values.get('HELIUM_PATCH', '0'),
+                            values.get('HELIUM_PLATFORM', '0'))
 
 
 def CompressUsingLZMA(build_dir,
@@ -724,7 +718,7 @@ def _ParseOptions():
         '{BSDIFF|ZUCCHINI}.')
     parser.add_option('-n',
                       '--output_name',
-                      default='chrome',
+                      default='helium',
                       help='Name used to prefix names of generated archives.')
     parser.add_option('--enable_hidpi',
                       default='0',
--- a/chrome/version.gni
+++ b/chrome/version.gni
@@ -19,7 +19,9 @@
 # all values we need at once.
 _version_dictionary_template = "full = \"@MAJOR@.@MINOR@.@BUILD@.@PATCH@\" " +
                                "major = \"@MAJOR@\" minor = \"@MINOR@\" " +
-                               "build = \"@BUILD@\" patch = \"@PATCH@\" "
+                               "build = \"@BUILD@\" patch = \"@PATCH@\" " +
+                               "helium_full = \"@HELIUM_MAJOR@.@HELIUM_MINOR@" +
+                               ".@HELIUM_PATCH@.@HELIUM_PLATFORM@\" "
 
 # The file containing the Chrome version number.
 chrome_version_file = "//chrome/VERSION"
@@ -160,6 +162,7 @@ _result = exec_script("//build/util/vers
 
 # Full version. For example "45.0.12321.0"
 chrome_version_full = _result.full
+helium_version_full = _result.helium_full
 
 # The constituent parts of the full version.
 chrome_version_major = _result.major
--- a/components/policy/tools/generate_policy_source.py
+++ b/components/policy/tools/generate_policy_source.py
@@ -28,7 +28,7 @@ from xml.sax.saxutils import escape as x
 
 CHROME_POLICY_KEY = 'SOFTWARE\\\\Policies\\\\Google\\\\Chrome'
 CHROME_FOR_TESTING_POLICY_KEY = CHROME_POLICY_KEY + ' for Testing'
-CHROMIUM_POLICY_KEY = 'SOFTWARE\\\\Policies\\\\Chromium'
+CHROMIUM_POLICY_KEY = 'SOFTWARE\\\\Policies\\\\Helium'
 PLATFORM_STRINGS = {
     'chrome_frame': ['win'],
     'chrome_os': ['chrome_os'],
--- a/components/policy/tools/template_writers/writer_configuration.py
+++ b/components/policy/tools/template_writers/writer_configuration.py
@@ -27,21 +27,21 @@ def GetConfigurationForBuild(defines):
   if '_chromium' in defines:
     config = {
         'build': 'chromium',
-        'app_name': 'Chromium',
+        'app_name': 'Helium',
         'doc_url': 'https://chromeenterprise.google/policies/',
         'frame_name': 'Chromium Frame',
         'os_name': 'ChromiumOS',
         'webview_name': 'Chromium WebView',
         'win_config': {
             'win': {
-                'reg_mandatory_key_name': 'Software\\Policies\\Chromium',
+                'reg_mandatory_key_name': 'Software\\Policies\\Helium',
                 'reg_recommended_key_name':
-                'Software\\Policies\\Chromium\\Recommended',
+                'Software\\Policies\\Helium\\Recommended',
                 'mandatory_category_path': ['chromium'],
                 'recommended_category_path': ['chromium_recommended'],
                 'category_path_strings': {
-                    'chromium': 'Chromium',
-                    'chromium_recommended': 'Chromium - {doc_recommended}',
+                    'chromium': 'Helium',
+                    'chromium_recommended': 'Helium - {doc_recommended}',
                 },
                 'namespace': 'Chromium.Policies.Chromium',
             },
--- a/chrome/installer/setup/last_breaking_installer_version.cc
+++ b/chrome/installer/setup/last_breaking_installer_version.cc
@@ -11,12 +11,8 @@ namespace installer {
 // When updating the documentation, keep a history of maximum 3 milestones prior
 // the current version because downgrades are supported for 3 milestones.
 //
-// Breaking changes from 85.0.4169.0:
-//  Change: Default installation directory for fresh 64 bits browsers moved from
-//    base::DIR_PROGRAM_FILESX86 to DIR_PROGRAM_FILES.
-//  Reason for being breaking: Downgrading to previous version will result in
-//    stale data in DIR_PROGRAM_FILES since the previous installers do not know
-//    if there is Chrome data at this location.
-const wchar_t kLastBreakingInstallerVersion[] = L"85.0.4169.0";
+// Helium uses its own version scheme, which is below every Chromium
+// "breaking installer version".
+const wchar_t kLastBreakingInstallerVersion[] = L"0.0.0.0";
 
 }  // namespace installer
